code/modules/reagents/reagent_dispenser.dm

```diff
@@ -169,7 +169,7 @@
 		explode()
 
 /obj/structure/reagent_dispensers/fueltank/bullet_act(var/obj/item/projectile/Proj)
-	if(istype(Proj ,/obj/item/projectile/beam)||istype(Proj,/obj/item/projectile/bullet)||istype(Proj,/obj/item/projectile/ricochet))
+	if(istype(Proj ,/obj/item/projectile/beam)||istype(Proj,/obj/item/projectile/bullet)||istype(Proj,/obj/item/projectile/ricochet)&&!istype(Proj,/obj/item/projectile/ricochet/taser))
```

Comment:
Do you really need a hardcoded taser check here?

Reply:
Fuel tanks blow up without it, since it's just checking for _any_ ricochet projectile hitting it before exploding. It's quick and dirty though, there's probably a better way.

Reply:
Setting the damage to 0 should be enough, it checks for get_damage
Since you set the taser projectile to 0 damage you should try again without the check

Reply:
Yep, you're right, didn't look over the code enough. Tested, new commit added